prboom2/src/p_setup.c

```diff
@@ -1779,13 +1781,22 @@ static void P_LoadUDMFThings(int lump)
       mt.options |= MTF_AMBUSH;
 
     if (dmt->flags & UDMF_TF_SINGLE)
+    {
       mt.options |= MTF_GSINGLE;
+      mt.options &= ~MTF_NOTSINGLE;
```

Comment:
Will this change affect compatibility? Have you tested UDMF DSDA maps? I think I saw a few.
It's not a problem if there are no demos for such maps though.